horreum-backend/src/main/resources/application.properties

```diff
@@ -110,6 +110,8 @@ quarkus.log.category."org.hibernate.cache.internal".level=ERROR
 quarkus.log.category."org.hibernate.engine.internal".level=ERROR
 quarkus.log.category."org.hibernate.event.internal".level=ERROR
 quarkus.log.category."org.hibernate.internal".level=ERROR
+# This will remove the annoying HHH000444 log messages
```

Comment:
Looking at this issue, it was supposed to be fixed in `6.2.3.Final`, but we are using `6.2.7.Final`. Are we certain this isn't a genuine issue?